wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -100,6 +100,14 @@ function arcsiStop(episodeName) {
     });
 }
 
+function arcsiDownload(episodeName) {
+    gtag('event', 'Arcsi download', {
+        'event_category': episodeName,
+        'event_label': 'Download name',
+        'value': 1,
+    });
+}
+
```

Comment:
Isn't it redundant with the other PR (https://github.com/mmmnmnm/lahmacun/pull/387/files)?

Reply:
Yes, it is still the old script

Reply:
Ok but why does it appear here as change?